tests/dummy/app/pods/docs/guides/header/scroll-indicators/controller.js

```diff
@@ -63,6 +63,8 @@ export default Controller.extend({
   }),
 
   footerRows: computed(function() {
-    return generateRows(1);
+    return generateRows(100, 1, (row, key) => {
+      return String.fromCharCode(key.charCodeAt(0) + 7);
+    });
```

Comment:
By making this change here, have we lost test coverage for cases where the footer rows are *not* scrollable?

Reply:
_This_ change only affects the example in the docs. We could include an example for each case (single footer row vs. many scrollable), but this effectively demonstrates everything the bottom indicator does.

It's fair to ask the same question of the tests, however. I'll make sure both are covered explicitly.

Reply:
Done.